manager/state/raft/membership/cluster.go

```diff
@@ -180,7 +181,7 @@ func (c *Cluster) ReplaceMemberConnection(id uint64, oldConn *Member, newConn *M
 		return ErrIDNotFound
 	}
 
-	if oldConn.Conn != oldMember.Conn {
+	if !force && oldConn.Conn != oldMember.Conn {
```

Comment:
They are always in sync. In the case where the address doesn't change, the old address is passed in, so setting it is a no-op.